packages/api-plugin-promotions/src/qualifiers/stackable.js

```diff
@@ -0,0 +1,27 @@
+import { createRequire } from "module";
+import Logger from "@reactioncommerce/logger";
+
+const require = createRequire(import.meta.url);
+const pkg = require("../../package.json");
+
+const { name, version } = pkg;
+const logCtx = {
+  name,
+  version,
+  file: "stackable.js"
+};
+
+/**
+ * @summary does promotion meet stackability requirements
+ * @param {Object} context - The application context
+ * @param {Array<Object>} appliedPromotions - The promotions already applied
+ * @param {Object} promotion - The promotions we are trying to apply
+ * @return {{reason: string, qualifies: boolean}} - If it qualifies and if it doesn't why not
+ */
+export default function stackable(context, appliedPromotions, promotion) {
+  if (appliedPromotions[0].stackAbility === "none" || promotion.stackAbility === "none") {
+    Logger.info(logCtx, "Cart disqualified from promotion because stack ability is none");
+    return { qualifies: false, reason: "Cart disqualified from promotion because stack ability is none" };
```

Comment:
Just my personal view point-

Is the current "reason" message really required/adding value, since it is evident that false means promotions are not stackable.

"reason" would make sense if we are passing explicit/separate messages like:

if (appliedPromotions[0].stackAbility === "none") return { qualifies: false, reason: "A non stackable promotion is already applied to the cart and new promotions cannot be applied" };

if (promotion.stackAbility === "none") return { qualifies: false, reason: "The applied promotion does not support stackability" };

Reply:
No, incorrect. This might apply to stackability but there are other systems that may apply qualifiers that need to supply a reason that need to show up in the cart.

Reply:
Also, even stackability is going to have several types which will have separate messages